sphinx/source/start_network.rst

```diff
@@ -1,135 +1,102 @@
+To start up a network, the first node of the network should be started with the ``start`` option:
 
 .. code-block:: bash
 
-    $ cchost --enclave-file=/path/to/application --node-address=node_ip:node_port --rpc-address=rpc_ip:rpc_public_ip
-    --ledger-file=/path/to/ledger --node-cert-file=/path/to/node_certificate --quote-file=/path/to/quote
-    2019-08-06 15:04:36.951158        [info ] ../src/host/main.cpp:240             | Starting new node
-    2019-08-06 15:04:39.355423        [info ] ../src/host/main.cpp:257             | Created new node
-    ...
-    2019-08-06 15:04:40.542079        [info ] ../src/host/enclave.h:174            | Quote verified
-    ...
+    $ cchost --enclave-file=/path/to/enclave_library --enclave-type=debug
+    --node-address=node_ip:node_port --rpc-address=rpc_ip:rpc_port
+    --public-rpc-address=public_rpc_ip:public_rpc_port --ledger-file=/path/to/ledger
+    --node-cert-file=/path/to/node_certificate --quote-file=/path/to/quote
+    start --network-cert-file=/path/to/network_certificate --gov-script=/path/to/lua/governance_script
+    --member-certs=member_certificates_glob --user-certs=user_certificates_glob
 
-When starting up, each node generates its own key pair and outputs the certificate associated with the public key at the location specified by ``--node-cert``. A quote file, required for remote attestation when this node joins the network, is also output at the location specified by ``--quote-file``.
+When starting up, the node generates its own key pair and outputs the certificate associated with its public key at the location specified by ``--node-cert-file``. A quote file, required for remote attestation, is also output at the location specified by ``--quote-file``. The certificate of the freshly-created CCF network is also output at the location specified by ``--network-cert-file``.
 
-Configuring the initial state of the network
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The :ref:`governance` rules are defined as a Lua script passed via the ``--gov-script`` option. For example, a default set of `governance rules <https://github.com/microsoft/CCF/blob/master/src/runtime_config/gov.lua>`_ can be used to define a majority of members as the quorum of the consortium.
 
-Once the initial set of nodes is running, the ``nodes.json`` file specifying the configuration of the original network should be created. For example, for a network of two nodes, the ``nodes.json`` file will be:
+The identities of members and users are specified as `glob patterns <https://en.wikipedia.org/wiki/Glob_(programming)>`_ via the ``--member-certs`` and ``--user-certs`` option, respectively. For example, if 2 members (``member1_cert.pem`` and ``member2_cert.pem``) and 3 users (``user1_cert.pem``, ``user2_cert.pem`` and ``user3_cert.pem``) should be added to CCF, operators should specify ``--member-certs=member*_cert.pem`` and ``--user-certs=user*_cert.pem``.
```

Comment:
Is this _should_ specify or _may_ specify? Do we support `--member-certs=member_cert1.pem --member-certs=member_cert2.pem` when we want to be explicit? (The naming patterns work for our test keys but do we actually require operators to dump the keys with globbable names?)

Reply:
_should_ for now. I will create a story to fix this as globbing is a little awkward.

Reply:
I think we should remove the globbing functionality from CCF. Shells do globbing just fine, there's no reason to have it.

Reply:
See https://github.com/microsoft/CCF/issues/334 for follow up work.